crmsh/bootstrap.py

```diff
@@ -1844,6 +1844,37 @@ def update_nodeid(nodeid, node=None):
         utils.disable_service("corosync-qdevice.service")
 
 
+def adjust_priority_in_rsc_defaults(is_2node_wo_qdevice):
+    """
+    Adjust priority in rsc_defaults
+
+    Set priority=1 when current cluster is 2 nodes without qdevice;
+    else set priority=0
+    """
+    if is_2node_wo_qdevice:
+        utils.set_property("priority", 1, property_type="rsc_defaults", property_id="rsc-options", conditional=True)
+    else:
+        utils.set_property("priority", 0, property_type="rsc_defaults", property_id="rsc-options")
```

Comment:
Does it mean we always want it to be put into "rsc-options" meta set, no matter if there's any other existing set? Wondering what is the consideration here.

Reply:
Thanks for remind me this.

I changed inside `set_property`, check if property_id exists, if not, set the property_id as None, to use the default one